### PR TITLE
JADE: Character Generation

### PR DESCRIPTION
**Pull request by Nostritius**
This PR tries to implement as much as possible the Jade Empire character generation menu and generates the player character.

### CODE REVIEW

File: src/engines/jade/gui/chargen/rules.mk [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# You should have received a copy of the GNU General Public License
+# along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+
+# Character generation in Jade Empire.
+
+src_engines_jade_libjade_la_SOURCES += \
+    $(EMPTY)
+
+src_engines_jade_libjade_la_SOURCES += \
+    $(EMPTY)
```
Comment (DrMcCoy) [inline src/engines/jade/gui/chargen/rules.mk]: Missing a new line at the end of the file

**Comment by Nostritius** [comment]
The issues should now be fixed. About the exception, you mentioned, it's about the background of the main menu. While i fixed the error for the character generation, this will almost certainly make more problems for example, when going back to the main menu from the game.

**Comment by DrMcCoy** [comment]
Now starting the game doesn't work anymore. It just returns to back the menu when it should display the first area.

**Comment by Nostritius** [comment]
Ok, now it should work

**Comment by DrMcCoy** [comment]
Yeah, now it does

Merged, thanks! :)

**Pull request merged**